gulpfile.js

```diff
@@ -1,137 +1,8 @@
 'use strict';
 
-const $ = require('gulp-load-plugins')();
-const del = require('del');
-const gulp = require('gulp');
-const manifest = require('./package.json');
-const path = require('path');
-const spawnSync = require('child_process').spawnSync;
-
-const coverageDir = path.join(__dirname, 'coverage');
-const distDir = path.join(__dirname, 'dist');
-const docsDir = path.join(__dirname, 'docs');
-
-/*
- * Clean tasks
- */
-gulp.task('clean', ['clean-coverage', 'clean-dist', 'clean-docs']);
-
-gulp.task('clean-coverage', done => { del([coverageDir]).then(() => done()) });
-
-gulp.task('clean-dist', done => { del([distDir]).then(() => done()) });
-
-gulp.task('clean-docs', done => { del([docsDir]).then(() => done()) });
-
-/*
- * build tasks
- */
-gulp.task('build', ['clean-dist', 'lint-src'], () => {
-	return gulp
-		.src('src/**/*.js')
-		.pipe($.plumber())
-		.pipe($.debug({ title: 'build' }))
-		.pipe($.babel({
-			sourceMaps: 'inline'
-		}))
-		.pipe(gulp.dest(distDir));
+require('appcd-gulp')({
+	gulp:     require('gulp'),
+	pkgJson:  require('./package.json'),
+	template: 'standard',
+	babel:    'node4'
```

Comment:
Please bump this to `node6`. We're not supporting Node 4 and older for this type of library.